doc/MAGIC.md

```diff
@@ -922,7 +923,8 @@ A small subset of enchantments can be applied to monsters via effects. These are
 
 Character status value  | Description
 ---                     |---
-`ARMOR_ACID`            | Negative values give armor against the damage, positive values make the monster accept more damage of this type.
+`ARMOR_ALL`             | Negative values give armor against the damage, positive values make the monster accept more damage of this type.
```

Comment:
ARMOR_ALL actually comes after ARMOR_ACID alphabetically.

Reply:
Alphabetized this and magic_enchant.h